wgpu-core/src/command/render.rs

```diff
@@ -387,10 +387,22 @@ pub enum RenderPassErrorInner {
     Encoder(#[from] CommandEncoderError),
     #[error("attachment texture view {0:?} is invalid")]
     InvalidAttachment(id::TextureViewId),
-    #[error("there are attachments")]
-    NoAttachments,
-    #[error("attachments have different sizes")]
-    MismatchAttachments,
+    #[error("necessary attachments are missing")]
+    MissingAttachments,
+    #[error("color and/or depth attachments have differing sizes: a {mismatching_attachment_type_name} attachment of size {mismatching_dimensions_width}x{mismatching_dimensions_height} is not compatible with a {previous_attachment_type_name} attachment's size ({previous_dimensions_height}x{previous_dimensions_width})",
```

Comment:
just printing the tuple in the error would be nearly as clear, and would save us a few keystrokes here. Perhaps, we can avoid those `previous_dimensions_width = ...` defines, and just print the whole tuples?